### PR TITLE
Updated instagram media modal regex and added unit test.

### PR DESCRIPTION
**Pull request by jalopez13**
Issue: https://github.com/triniti/cms-js/issues/51

Updated instagram media block regex and added unit test.

### CODE REVIEW

File: src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx [hunk 1]
```diff
@@ -104,16 +105,17 @@ export default class InstagramMediaBlockModal extends React.Component {
     let { errorMsg, hideCaption, id, isValid } = this.state;
     const input = event.target.value;
     const isValidUrl = INSTAGRAM_MEDIA_REGEX.test(input);
-
-    if (!isValidUrl) {
-      errorMsg = 'url or embed code is invalid';
-      isValid = false;
-    } else {
+    console.log('ID: ', input.match(INSTAGRAM_STRIP_REGEX)[0]);
```
Comment (efjacobson) [inline src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx]: @jalopez13 remove log

File: src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx [hunk 2]
```diff
@@ -50,7 +51,7 @@ export default class InstagramMediaBlockModal extends React.Component {
       isValid: block.has('id'),
       touched: false,
       updatedDate: block.get('updated_date', new Date()),
-      url: id ? `https://www.instagram.com/p/${id}/` : '',
+      url: '',
```
Comment (efjacobson) [inline src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx]: @jalopez13 why? this should show the url when opening modal to edit existing block

File: src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx (hunk 1 above)
Comment (jalopez13) [reply src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx]: @efjacobson 

Removed!

File: src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx (hunk 2 above)
Comment (jalopez13) [reply src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx]: @efjacobson 

Added `id ? `https://www.instagram.com/p/${id}/` : ''` that shouldn't have been removed.

File: src/plugins/blocksmith/components/instagram-media-block-modal/index.test.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import test from 'tape';
+import INSTAGRAM_MEDIA_REGEX from './instagramMediaBlockRegex';
+
+test('InstagramMediaBlock', (t) => {
```
Comment (efjacobson) [inline src/plugins/blocksmith/components/instagram-media-block-modal/index.test.jsx]: @jalopez13 this isnt testing index so it should not be called `index.test.jsx`. name it after the file it is testing

File: src/plugins/blocksmith/components/instagram-media-block-modal/index.test.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```
Comment (efjacobson) [inline src/plugins/blocksmith/components/instagram-media-block-modal/index.test.jsx]: @jalopez13 unnecessary

File: src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx [hunk 5]
```diff
@@ -131,12 +133,14 @@ export default class InstagramMediaBlockModal extends React.Component {
...
       hasUpdatedDate,
       hideCaption,
+      id,
       isValid,
       touched,
       updatedDate,
       url,
     } = this.state;
     const { isFreshBlock, isOpen, toggle } = this.props;
+    const displayUrl = isValid ? `https://www.instagram.com/p/${id}/` : url;
```
Comment (efjacobson) [inline src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx]: @jalopez13 doesn't `url` already have what you need?

File: src/plugins/blocksmith/components/instagram-media-block-modal/index.test.jsx (hunk 3 above)
Comment (jalopez13) [reply src/plugins/blocksmith/components/instagram-media-block-modal/index.test.jsx]: @efjacobson 

Renamed to `instagramMediaBlockRegex.test.js`.

File: src/plugins/blocksmith/components/instagram-media-block-modal/index.test.jsx (hunk 4 above)
Comment (jalopez13) [reply src/plugins/blocksmith/components/instagram-media-block-modal/index.test.jsx]: @efjacobson 

Removed!

File: src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx (hunk 5 above)
Comment (jalopez13) [reply src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx]: @efjacobson 

Updated to use url by itself.

**Pull request merged**